modules/programs/less/default.nix

```diff
@@ -0,0 +1,75 @@
+{ config, mode, pkgs, lib, ... }:
+
+with lib;
+
+# TODO: submit this module upstream to home-manager
+let
+  cfg = config.soxin.programs.less;
+
+  keyboardLayout = config.soxin.settings.keyboard.defaultLayout.console.keyMap;
+
+  colemakKeybindings = ''
+    #command
+    n left-scroll
+    e forw-line
+    i back-line
+    o right-scroll
+    k repeat-search
+    K reverse-search
+  '';
+
+  bepoKeybindings = ''
+    #command
+    t left-scroll
+    s forw-line
+    r back-line
+    n right-scroll
+    ' repeat-search
+    ? reverse-search
+  '';
+
+in
+{
+  options.soxin.programs.less = {
```

Comment:
Don't we want to use `mkSoxinModule` to be able to override the keyboard layout here?

Reply:
done